src/commands/distribute/releases/show.ts

```diff
@@ -61,7 +61,7 @@ export default class ShowReleaseDetailsCommand extends AppCommand {
       ["Download URL", "downloadUrl"],
       ["Install URL", "installUrl"],
       ["Icon URL", "appIconUrl"],
-      ["Distribution Group", "distributionGroups", (distributionGroups: models.DistributionGroup[]) => _.get(distributionGroups, "[0].name", "")]
+      ["Destinations", "destinations", (destinations: models.Destination[]) => destinations && destinations.length > 1 ? JSON.stringify(destinations, null, 2) : noDestinations]
```

Comment:
Just out of curiosity: We are printing the destinations by using `JSON.stringify()` here. Is that a common pattern in the CLI? I would have expected a more reader-friendly format here.

Reply:
@pfleidi the current out.ts formatting options don't handle lists of objects particularly well.  Currently there is `out.report.allAttributes` which doesn't go further than one level, and `out.report.inspect` which doesn't do any delimiter formatting.

There also seems to be some report formatting that overrides any new line indentation.

At this point we considered adding a stringify wrapper to out.ts like inspect, but thought against it.